src/solver/tron.jl

```diff
@@ -180,7 +180,7 @@ function projected_line_search!{T <: Real}(x::AbstractVector{T},
   end
 
   project_step!(s, x, d, ℓ, u, α)
-  project!(x, x .+ s, ℓ, u)
+  x .= x .+ s
```

Comment:
The matlab code had
```
s = gpstep(x, alpha, w, xl, xu)
x = mid(x + alpha * w, xl, xu)
```
but that means that `s = P(x + alpha * w) - x = xnew - x`, so I simplified. Unless it was done for numerical reasons.